modules/tinymce/src/core/test/ts/browser/FormatterApplyTest.ts

```diff
@@ -225,7 +225,7 @@ UnitTest.asynctest('browser.tinymce.core.FormatterApplyTest', function (success,
     rng.setEnd(editor.dom.select('p')[0].firstChild, 4);
     editor.selection.setRng(rng);
     editor.formatter.apply('format');
-    LegacyUnit.equal(getContent(editor), '<p><b style=\"color: #ff0000; font-size: 10px;\">1234</b></p>', 'Inline element with styles');
+    LegacyUnit.equal(getContent(editor), '<p><b><span style="color: #ff0000; font-size: 10px;">1234</span></b></p>', 'Inline element with styles');
```

Comment:
This seems like it would be an issue, as it goes against what the formatter declared. That means that the formatter won't match, so you won't be able to toggle it anymore with this change.

Reply:
Thanks for picking that up! I forgot about the ability to toggle formats. That definitely complicates things.